content/source/docs/enterprise/before-installing/centos-requirements.html.md

```diff
@@ -12,8 +12,8 @@ When installing Terraform Enterprise on CentOS Linux, ensure your OS and Docker
 * A [supported version](/docs/enterprise/before-installing/index.html#operating-system-requirements) of CentOS.
 * A suitable version of Docker:
    * Docker 1.13.1 (available in the `extras` repository).
-   * [Docker CE](https://docs.docker.com/install/linux/docker-ce/centos/) version 17.06 through 18.x (Docker v19.x is currently unsupported.)
-   * [Docker EE](https://docs.docker.com/install/linux/docker-ee/centos/) version 17.06 through 18.x (Docker v19.x is currently unsupported.)
+   * [Docker CE](https://docs.docker.com/install/linux/docker-ce/centos/) version 17.06 through 19.x
```

Comment:
I'd like to get @amy-hashi's opinion on this suggestion since the current maximum for Replicated is `19.03.8`. 

```suggestion
   * [Docker CE](https://docs.docker.com/install/linux/docker-ce/centos/) version 17.06 through 19.03.8.
```